File: Star Patterns/ReversedPyramids/solidReversedPyramid/solidReversedPyramid.java
class solidReversedPyramid {
	public static void main (String[] args) {
        int numRows = 7;
        int temp = (numRows * 2) - 1; // temp = 13 in this case
        int colVal;
        int rowVal;
        char patternSymbol = '*';

        for(rowVal = 1; rowVal <= numRows; rowVal++) {
            for(colVal = 1; colVal < rowVal; colVal++) {
                System.out.print("  ");
            } 
            for(colVal = 1; colVal <= temp; colVal++) {
                System.out.print(" " + patternSymbol); // print() function for avoiding line break
            }
            temp -= 2;
            System.out.println(""); // println() For including line break after printing each row.
        }
    }
}

File: Star Patterns/ReversedPyramids/solidReversedPyramid/solidReversedPyramid.py
numRows = 7
temp = (numRows * 2) - 1
patternSymbol = '*'

for rowVal in range(1, numRows + 1):
    for colVal in range(1, rowVal + 1):
        print("  ", end="")

    for colVal in range(1, temp + 1):
        print(patternSymbol, "", end="") # heer, I am only using this "" because, sep parameter is already giving us one space as sep=" "
    
    temp -= 2;  
    print()   # for new line